### PR TITLE
add rb-dotenv to conda-forge

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,8 +1,3 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
-# Jinja variables help maintain the recipe as you'll update the version only here.
-# Using the name variable with the URL in line 14 is convenient
```
Comment (bgruening) [inline recipes/example/meta.yaml]: please do not change those files, only add your recipe :)

Thanks.

File: recipes/rb-dotenv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: {{ name }}-{{ version }}/LICENSE
+  summary: 'Ruby gem to load environment variables from `.env`'
+  doc_url: https://www.rubydoc.info/gems/dotenv/2.7.5
+  dev_url: https://github.com/bkeepers/dotenv
+
+extra:
+  recipe-maintainers:
+    - rluria14
```
Comment (bgruening) [inline recipes/rb-dotenv/meta.yaml]: You maybe want to add @sodre who is currently spearheading the ruby package builds.

**Comment by sodre** [comment]
Hi @rluria14, if possible can you start the recipe from the template I setup [here](https://github.com/conda-forge/staged-recipes/pull/9320/files)

File: recipes/rb-dotenv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  home: https://rubygems.org/gems/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: {{ name }}-{{ version }}/LICENSE
+  summary: 'Ruby gem to load environment variables from `.env`'
+  doc_url: https://www.rubydoc.info/gems/dotenv/2.7.5
+  dev_url: https://github.com/bkeepers/dotenv
+
+extra:
+  recipe-maintainers:
```
Comment (sodre) [inline recipes/rb-dotenv/meta.yaml]:
```suggestion
  recipe-maintainers:
    - sodre
```

File: recipes/rb-dotenv/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    - ruby -r {{ name }} -e 'exit 0'
+    - dotenv --help
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: {{ name }}-{{ version }}/LICENSE
+  summary: 'Ruby gem to load environment variables from `.env`'
+  doc_url: https://www.rubydoc.info/gems/dotenv/2.7.5
```
Comment (sodre) [inline recipes/rb-dotenv/meta.yaml]:
```suggestion
  doc_url: https://www.rubydoc.info/gems/{{ name }}
```

File: recipes/dash-cytoscape/meta.yaml [hunk 5]
```diff
@@ -1,40 +0,0 @@
-{% set name = "dash_cytoscape" %}
```
Comment (bgruening) [inline recipes/dash-cytoscape/meta.yaml]: @rluria14 you still removed this one here ...

**Comment by rluria14** [comment]
> Undo the changes outside the rb-dotenv folder, and add me as a maintainer.

Changes have been made. Thanks.

File: recipes/example/meta.yaml [hunk 6]
```diff
@@ -79,4 +79,7 @@ extra:
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
     - LisaSimpson
     - LandoCalrissian
+<<<<<<< HEAD
```
Comment (bgruening) [inline recipes/example/meta.yaml]: @rluria14 if you check the github interface you should be able to see these mistakes here.

This PR should only contain you on meta.yaml file.

**Pull request closed without merging**